src/antelope/stores/chain.ts

```diff
@@ -144,6 +156,26 @@ export const useChainStore = defineStore(store_name, {
                 useFeedbackStore().unsetLoading('updateApy');
             }
         },
+        async updateGasPrice(label: string) {
+            useFeedbackStore().setLoading('updateGasPrice');
+            this.trace('updateGasPrice');
+            const chain = this.getChain(label);
+            try {
+                if (!chain.settings.isNative()) {
+                    const gasPrice = await (chain.settings as EVMChainSettings).getGasPrice();
+                    const wei = gasPrice.toNumber();
+                    const gwei = (wei / 1000000000).toFixed(2);
```

Comment:
since wei needs to be bignumber, this will need to change to `wei.div(1000000000).toNumber.toFixed(2)` i think